custom_components/tuya_local/devices/atorch_s1wp.yaml

```diff
@@ -531,3 +531,15 @@ secondary_entities:
             value: "Off"
           - dps_val: memory
             value: Memory
+  - entity: number
+    category: config
+    name: Update interval
+    icon: "mdi:update"
+    dps:
+      - id: 139
+        type: integer
```

Comment:
I think this requires `optional: true`. DP 139 does not appear in the log posted to #1103

Reply:
added

Reply:
is it already available on the release?

Reply:
> oh... so i have to use automation to prevent it going back to original 60 seconds, right?

yes, you can make trigger like this:
```
platform: state
entity_id:
  - number.ju_wei_dian_neng_bao_ji_kong_xing_s1wp_update_interval
to: "60"
from: null
```